spec/acceptance/ini_subsetting_spec.rb

```diff
@@ -1,19 +1,19 @@
 require 'spec_helper_acceptance'
 
-tmpdir = default.tmpdir('tmp')
-
 describe 'ini_subsetting resource' do
+  basedir = setup_test_directory
+
   after :all do
-    shell("rm #{tmpdir}/*.ini", acceptable_exit_codes: [0, 1, 2])
+    run_shell("rm #{setup_test_directory}/*.ini", expect_failures: true)
```

Comment:
Would this not execute the setup of the test directory as opposed to referring to the path already returned in basedir?